guidelines.md

```diff
@@ -101,3 +101,61 @@ As an example, consider minimum and maximum length of the reservation:
 * MinLength `P0Y0M1DT0H0M0S`
 * MaxLength `P0Y0M3DT0H0M0S`
 
+# Mews Certification
+
+Before going live in the production environment all connections must be certified by Mews.
+
+### Pre-Certification
+
+- Sign in to the Mews Connector API demo account. Credentials for this can be found in the [guidelines](https://mews-systems.gitbook.io/connector-api/guidelines#authentication) section.
```

Comment:
the link should correspond to article it refers to

Reply:
I have adjusted the name to "Authentication" to match the link. It will now send users to the Authentication under the Guidelines section.